pkgs/development/tools/misc/seer/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, cmake, qtcharts, qtbase, wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "seer";
+  version = "1.7";
+
+  src = fetchFromGitHub {
+    owner = "epasveer";
+    repo = "seer";
+    rev = "v${version}";
+    sha256 = "sha256-/EuXit1kHW2cdqa5BJEj29Wu3WafVZb6DpPnIg2tDP0=";
+  };
+
+  sourceRoot = "source/src";
```

Have you tried `cd` before configure?

```suggestion
  preconfigure = ''
    cd src
  '';
```

---

I have not tried that, mostly because in-source builds are considered bad form in CMake projects. The package builds anyway so I can make this change.

---

This is not an "in-source build" because Nixpkgs takes care of it, automatically creating an empty dir.